### PR TITLE
testdays prep

### PR DESCRIPTION
**Pull request by rainleander**

### CODE REVIEW

File: source/testday/ussuri/milestone1.html.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
 ---
-title: RDO Stein Test Day
+title: RDO Ussuri Test Day
 ---
 
-# Stein Test Day 1
+# Stein Milestone1 Test Day
```
Comment (amoralej) [inline source/testday/ussuri/milestone1.html.md]: I guess that should be Ussuri Milestone1

File: source/testday/ussuri/milestone1.html.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-* [Workarounds](https://etherpad.openstack.org/p/rdo-test-days-stein1-workarounds)
+* [Workarounds](https://etherpad.openstack.org/p/rdo-test-days-ussuri1-workarounds)
 
 ## Prerequisites
 
 We plan to have packages for the following platforms:
 
-* [RHEL 7](https://access.redhat.com/products/red-hat-enterprise-linux/)
-* [CentOS 7](https://www.centos.org/download/)
+* [RHEL 8](https://access.redhat.com/products/red-hat-enterprise-linux/)
```
Comment (amoralej) [inline source/testday/ussuri/milestone1.html.md]: I like the idea of having a testday short after having it ready on CentOS8 but as today, we can't commit with having RDO on CentOS8 by Ussuri Milestone 1. We have external dependencies and we'll need some time to have it ready.

File: source/testday/ussuri/milestone1.html.md (hunk 1 above)
Comment (rainleander) [reply source/testday/ussuri/milestone1.html.md]: Good catch. Fixed!

File: source/testday/ussuri/milestone1.html.md (hunk 2 above)
Comment (rainleander) [reply source/testday/ussuri/milestone1.html.md]: Fair. Keep it on RHEL / CentOS 8 for Ussuri M3?

Comment (amoralej) [reply source/testday/ussuri/milestone1.html.md]: Hopefully at M3 we should have something for CentOS8.

**Pull request merged**